nifi-nar-bundles/nifi-hive-bundle/nifi-hive3-processors/src/main/java/org/apache/nifi/processors/hive/PutHive3Streaming.java

```diff
@@ -445,6 +449,10 @@ public void onTrigger(ProcessContext context, ProcessSession session) throws Pro
                 updateAttributes.put(HIVE_STREAMING_RECORD_COUNT_ATTR, Long.toString(hiveStreamingConnection.getConnectionStats().getRecordsWritten()));
                 updateAttributes.put(ATTR_OUTPUT_TABLES, options.getQualifiedTableName());
                 flowFile = session.putAllAttributes(flowFile, updateAttributes);
+                log.error(
+                        "Exception while processing {}. rollbackOnFailure='{}' - routing to failure. Exception: {}",
```

Comment:
Same comment as above, recommend removing `rollbackOnFailure` and `Exception: {}` references and pass the Exception to the `log.error()` method itself:

```
log.error("Error while processing {}, routing to failure", new Object[]{flowFile}, e);
```

Reply:
Applied all recommended changes.